packages/trpc/server/routers/viewer/routing-forms/response.schema.ts

```diff
@@ -6,7 +6,7 @@ export const ZResponseInputSchema = z.object({
   response: z.record(
     z.object({
       label: z.string(),
-      identifier: z.string().optional(),
```

Comment:
Ensure this property exists

Reply:
I think we should still keep it optional in this PR and later make it nullable in a separate PR as it might cause failure for certain embeds that are already loaded.